Microsoft.Packaging.Tools.Trimming/tasks/Microsoft.Packaging.Tools.Trimming.Tasks.csproj

```diff
@@ -70,6 +75,16 @@
   <ItemGroup>
     <PackageDependencyId Include="System.Reflection.Metadata" />
     <PackageDependencyId Include="System.Collections.Immutable" />
+    <PackageDependencyId Include="Newtonsoft.Json" />
```

Comment:
I'm not so happy with all these additions.  If I have time at some point I will rewrite the lock-file reading to just use json.net since this is what is done in other msbuild tasks (ResolveNuGetPackageAssets).